src/tools/OptionsSchema.json

```diff
@@ -0,0 +1,398 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Can we add a comment to this file? Or will it break some of the other tools? If we can, probably good to either repeat the content from readme.md, or point people to it.